CHANGELOG.md

```diff
@@ -46,6 +53,39 @@ Released: TBD (Not before 2025-05-01)
   generated for each of these removals.  Like merged class rules above, this
   should only be removing dead code.
   [#594](https://github.com/peggyjs/peggy/pull/594)
+- Character classes now process characters not in the Basic Multi-Lingual
```

Comment:
note that using a character not in the BMP automatically puts the character class in Unicode mode.